package.json

```diff
@@ -14,6 +14,7 @@
   },
   "homepage": "https://github.com/scality/Arsenal#readme",
   "dependencies": {
+    "ajv": "^4.1.3",
```

Comment:
Version `4.1.5` is out. Could we use a fixed version to promote build reproducibility and avoid random breakage?